docs/reference/env-variables.md

```diff
@@ -40,4 +40,21 @@ Environment variables which begin with `K3S_` will be preserved for the systemd
 
 Setting `K3S_URL` without explicitly setting an exec command will default the command to "agent".
 
-When running the agent, `K3S_TOKEN` must also be set.
\ No newline at end of file
+When running the agent, `K3S_TOKEN` must also be set.
+
+:::info Version Gate
+Available as of October 2024 releases: v1.28.15+k3s1, v1.29.10+k3s1, v1.30.6+k3s1, v1.31.2+k3s1.
+
+K3s will now use `PATH` to find runtimes, but it's necessary to add the `PATH` within the `k3s.service.env` or `k3s-agent.service.env` if you are using a agent.
+
+This example will add the `PATH` to `k3s.service.env` in the default directory:
+
+```bash
+echo PATH=$PATH >> /etc/systemd/system/k3s.service.env
```

Comment:
This file is overwritten by the install script whenever it is run, we should not recommend that users modify it because changes to it may be lost. We should instead recommend one of the other two env files:
```
EnvironmentFile=-/etc/default/k3s # or k3s-agent
EnvironmentFile=-/etc/sysconfig/k3s # or k3s-agent
```